docs/source/schema/unions-interfaces.md

```diff
@@ -1,48 +1,84 @@
 ---
 title: Unions and interfaces
-description: How to add unions and interfaces to a schema
+description: Polymorphic schema types
```

Comment:
I think in GraphQL our tendency is to call these "abstract" types rather than polymorphic. I'm open to being convinced otherwise, just sharing my experience.